pkg/gardenlet/controller/shoot/shoot_control_delete.go

```diff
@@ -96,37 +96,6 @@ func (c *Controller) runDeleteShootFlow(ctx context.Context, o *operation.Operat
 			}
 			return err
 		}),
-		// Check if Seed object for shooted seed has been deleted
```

Comment:
No, it's not needed, since we have the admission plugin to prevent shoot deletion of shoots that are referenced by managed seeds. That's why it's removed. Do you have any objections or concerns?

Reply:
Ah, okay, no, just wanted to clarify on this. Thanks!